MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -831,13 +831,21 @@ private void OnCreateAnimation(CreateAnimation payload)
             if (actor != null)
             {
                 actor.GetOrCreateActorComponent<AnimationComponent>()
-                    .CreateAnimation(payload.AnimationName, payload.Keyframes, payload.Events, payload.WrapMode, () =>
-                    {
-                        Protocol.Send(new OperationResult()
+                    .CreateAnimation(
+                        payload.AnimationName,
+                        payload.Keyframes,
+                        payload.Events,
+                        payload.WrapMode,
+                        payload.InitialState,
+                        isInternal: false,
+                        onCreatedCallback: () =>
                         {
-                            ResultCode = OperationResultCode.Success
-                        }, payload.MessageId);
-                    });
+                            Protocol.Send(new OperationResult()
+                            {
+                                ResultCode = OperationResultCode.Success
```

Comment:
Is it possible for this operation to fail? Do we catch it?

Reply:
Good catch. Wrapped in a try/catch.